_posts/2015-05-30-the-state-of-javascript.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Predictions on JavaScript in the next 12 months
+intro: Based on a recent talk I gave, in this blog post I'll discuss my thoughts on the current state of JS and what we might see over the next 12 months.
+---
+
+Recently I gave a talk at the [London JS
+Community](https://twitter.com/london_js)'s inaugural meetup titled "The State
+of JavaScript". You can find the slides for this below:
+
+<script async class="speakerdeck-embed" data-id="d15b87038dbc468ba94e31d0fef5118f" data-ratio="1.33333333333333" src="//speakerdeck.com/assets/embed.js"></script>
+
+In this post I'd like to focus specifically on the end of the talk, when I
+discuss my predictions for what we'll see happen in the next 12 months or so
+with JavaScript. Be warned this is quite opinionated, and I don't expect people to
+agree with everything I say! You should read this as "what Jack thinks" rather
+than "what will happen". Find me [on Twitter](http://twitter.com/Jack_Franklin)
+if you'd like to discuss things further.
+
+# Predictions
+
+I made 8 predictions on what I think we'll see in the next 12 months, and most
+of these are influenced by the three core goals of ECMAScript 2015, which hopes
+to provide a better language for:
+
+- complex applications
+- libraries
+- code generation (languages that compile to JS)
+
+### 1: Fewer people will write JavaScript without a compilation step
+
+We're seeing this trend already, libraries like
+[TypeScript](http://www.typescriptlang.org/) and [Babel](http://babeljs.io/)
+have built on what [CoffeeScript](http://coffeescript.org/) showed people
+wanted, by building on top of JavaScript and compiling down to JavaScript.
+CoffeeScript deserves a lot of credit here, it was the first project that really
```

Comment:
the clause before the comma is independent, and the following clause ellaborates, so I would use a colon instead of a comma. If you’d rather not, then it could be subbed out for either a full stop or a semi colon too.